README.md

```diff
@@ -26,13 +26,16 @@ const Equity = require('equityjs');
 
 const equity = new Equity({
   consumerKey: 'YOUR_CONSUMER_KEY',
-  consumerSecret: 'YOUR_CONSUMER_SECRET'
+  consumerSecret: 'YOUR_CONSUMER_SECRET',
+  username: 'user', //(optional)
+  password: 'pass', //(optional)
+  grant_type: 'password' //(optional)
```

Comment:
By passing in `username` and `password`, we are forcing the developer to create multiple instances of `Equity` for each user their application handles. That is inefficient and cumbersome. I believe there's a better way to handle this.